### PR TITLE
[DOCS] Added reference to Extend Config Data for the `Store` GraphQl …

### PR DESCRIPTION
**Pull request by lewisvoncken**
…endpoint

## Purpose of this pull request

This pull request (PR) adds a reference to add your own configuration to the `Store` GraphQl endpoint by using the extendConfigData array within the construct of the `StoreConfigDataProvider`

Added the following content:
image

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/graphql/reference/store-config.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php#L50

## Mage2gen example module

- https://mage2gen.com/load/a72e1739-76b1-469c-a33d-8ef39a5b9bed?main_version=3

whatsnew
Described how to add custom store configuration attributes to the [storeConfig query](https://devdocs.magento.com/guides/v2.3/graphql/reference/store-config.html).

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/reference/store-config.md [hunk 1]
```diff
@@ -80,6 +80,35 @@ Attribute |  Data Type | Description
 `no_route` | String | Contains the URL of the default page that you want to appear when if a 404 “Page not Found” error occurs
 `show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: `0` (No) or `1` (Yes)
 
+### Extend Config Data
+
+Easily add your own configuration to the `Store` endpoint within your own module.
+
+You can configure the constructor argument extendedConfigData in your area specific `etc/graphql/di.xml` in the argument node.
+
+The following example adds an array-item to the extendedConfigData array within the construct of the `StoreConfigDataProvider` .
```
Comment (ronak2ram) [inline guides/v2.3/graphql/reference/store-config.md]:
```suggestion
The following example adds an array-item to the extendedConfigData array within the construct of the `StoreConfigDataProvider`.
```

**Comment by lewisvoncken** [comment]
@ronak2ram I have just updated according to the suggested change.

**Comment by lewisvoncken** [comment]
@keharper I am willing to do this but in my opinion it should be separated from this pr so I will make a new one soon.

File: guides/v2.3/graphql/reference/store-config.md [hunk 2]
```diff
@@ -80,6 +80,35 @@ Attribute |  Data Type | Description
 `no_route` | String | Contains the URL of the default page that you want to appear when if a 404 “Page not Found” error occurs
 `show_cms_breadcrumbs` | Int | Determines if a breadcrumb trail appears on all CMS pages in the catalog. Options: `0` (No) or `1` (Yes)
 
+### Extend Config Data
```
Comment (keharper) [inline guides/v2.3/graphql/reference/store-config.md]:
```suggestion
### Extend configuration data
```

**Comment by keharper** [comment]
@lewisvoncken One more small change.

**Comment by lewisvoncken** [comment]
@keharper 
The small changes has been applied

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
Thanks for your contribution, @lewisvoncken .

**Pull request merged**